cinema/views.py

```diff
@@ -1,6 +1,7 @@
 from datetime import datetime
 
 from django.db.models import F, Count
+from drf_spectacular.utils import extend_schema, OpenApiParameter
```

Comment:
The import for `OpenApiParameter` is correct, but make sure that it is used consistently in the code. It seems to be used correctly in the `extend_schema` decorator.